modules/builds-running-entitled-builds-with-sharedsecret-objects.adoc

```diff
@@ -78,35 +111,41 @@ $ oc create rolebinding shared-resource-my-share --role=shared-resource-my-share
 apiVersion: build.openshift.io/v1
 kind: BuildConfig
 metadata:
-  name: my-csi-bc
-  namespace: my-csi-app-namespace
+  name: entitled-build
+  namespace: entitlement-test
 spec:
   runPolicy: Serial
   source:
     dockerfile: |
-      FROM registry.redhat.io/ubi9/ubi:latest
-      RUN ls -la /etc/pki/entitlement
+      FROM registry.access.redhat.com/ubi8/ubi:8.7-1112
       RUN rm /etc/rhsm-host
-      RUN yum repolist --disablerepo=*
-      RUN subscription-manager repos --enable rhocp-4.9-for-rhel-8-x86_64-rpms
-      RUN yum -y update
-      RUN yum install -y openshift-clients.x86_64
+      RUN yum --enablerepo=codeready-builder-for-rhel-8-x86_64-rpms \
+          install \
+          sudo \
+          procps-ng \
+          curl \
+          cronie \
+          crontabs \
+          nss_wrapper \
+          uid_wrapper -y && \
+          yum clean all -y
   strategy:
     type: Docker
     dockerStrategy:
+      from:
+        kind: DockerImage
+        name: registry.access.redhat.com/ubi8/ubi:8.7-1112
       volumes:
         - mounts:
             - destinationPath: "/etc/pki/entitlement"
-          name: my-csi-shared-secret
+          name: etc-pki-entitlement
           source:
-            csi:
-              driver: csi.sharedresource.openshift.io
-              readOnly: true
-              volumeAttributes:
-                sharedSecret: my-share-bc
-            type: CSI
+            secret:
+              defaultMode: 420
+              secretName: etc-pki-entitlement
+            type: Secret
```

Comment:
This is the sharedSecret doc, so I think we want to keep the existing CSI SharedResource example and not show the use of a standalone secret.